.github/workflows/tests.yml

```diff
@@ -0,0 +1,23 @@
+name: cargo-gccrs Unit and Functional tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      master
+
+jobs:
+  tests:
```

Comment:
I wonder should tests be called "integration tests" seems like it might be a better job name